rln/src/circuit.rs

```diff
@@ -1,22 +1,19 @@
 // This crate provides interfaces for the zero-knowledge circuit and keys
 
-use ark_bn254::{Bn254, Fq, Fq2, Fr, G1Affine, G1Projective, G2Affine, G2Projective};
-use ark_circom::{read_zkey, CircomBuilder, CircomConfig, WitnessCalculator};
-use ark_ff::BigInteger256;
+use ark_bn254::{
+    Bn254, Fq as ArkFq, Fq2 as ArkFq2, Fr as ArkFr, G1Affine as ArkG1Affine,
```

Comment:
This a change in functionality or so, belongs in a separate PR targeting an issue. Not in a release PR...